templates/Program/program_description.html.twig

```diff
@@ -57,6 +58,14 @@
           {{ "noCredits"|trans({}, "catroweb") }}
         {% endif %}
       </div>
+      <div id="credits-translation-wrapper" style="display: none;">
+        <div id="credits-translation" class="text-break"></div>
```

Comment:
I think you might want to separate the credit translation from the byline div. Even though the byline is conceptually part of the credit, it might be more elegant separating them.

Reply:
I originally did that, but then the animation gets split up and there are two things that open up which doesn't look as smooth as when the credit translation is within the credit-translation-wrapper and it all opens together

Reply:
ok